app/components/Entity/components/AnalysisMethod/components/Workflow/workflow.tsx

```diff
@@ -0,0 +1,51 @@
+import { SouthIcon } from "@databiosphere/findable-ui/lib/components/common/CustomIcon/components/SouthIcon/southIcon";
+import {
+  Loading,
+  LOADING_PANEL_STYLE,
+} from "@databiosphere/findable-ui/lib/components/Loading/loading";
+import { useAsync } from "@databiosphere/findable-ui/lib/hooks/useAsync";
+import { Grid2, Typography } from "@mui/material";
+import { Props } from "./types";
+import { TEXT_BODY_500 } from "@databiosphere/findable-ui/lib/theme/common/typography";
+import { getWorkflowLandingUrl } from "../../../../../../utils/galaxy-api";
+import {
+  ANCHOR_TARGET,
+  REL_ATTRIBUTE,
+} from "@databiosphere/findable-ui/lib/components/Links/common/entities";
+import { StyledGrid2 } from "./workflow.styles";
+import { TYPOGRAPHY_PROPS } from "../../constants";
+import { GRID2_PROPS, ICON_PROPS } from "./constants";
+
+export const Workflow = ({
+  geneModelUrl,
+  genomeVersionAssemblyId,
+  workflow,
+}: Props): JSX.Element => {
+  const { trsId, workflowDescription, workflowName } = workflow;
+  const { data: landingUrl, isLoading, run } = useAsync<string>();
+  return (
+    <StyledGrid2
+      {...GRID2_PROPS}
+      onClick={async (): Promise<void> => {
```

Comment:
The `onClick` function is unchanged from [AnalysisMethod component](https://github.com/galaxyproject/brc-analytics/blob/main/app/components/Entity/components/AnalysisMethod/analysisMethod.tsx#L29-L45).